service_crategen/src/commands/generate/codegen/rest_xml.rs

```diff
@@ -41,7 +41,7 @@ impl GenerateProtocol for RestXmlGenerator {
                         {set_parameters}
                         {build_payload}
 
-                        request.sign(&try!(self.credentials_provider.credentials()));
+                        request.sign_with_plus(&try!(self.credentials_provider.credentials()), true);
```

Comment:
This generates code different than the generated services committed in this PR.  I'll annotate an S3 example to show that.

Do we want this to be `, false`?

Reply:
Yes I did. I was messing around making sure failures propagated, and I forgot to change it back! :astonished: